clusters/base/kube-prometheus-stack.yaml

```
@@ -22,8 +22,36 @@ spec:
       sourceRef:
         kind: HelmRepository
         name: prometheus
+
   values:
+    alertmanager:
+      alertmanagerSpec:
+        nodeSelector:
+          cncf-project: wg-green-reviews
+          cncf-project-sub: internal
+
     prometheus:
       prometheusSpec:
         serviceMonitorSelectorNilUsesHelmValues: false
+
+      nodeSelector:
+        cncf-project: wg-green-reviews
+        cncf-project-sub: internal
```

```suggestion

        nodeSelector:
          cncf-project: wg-green-reviews
          cncf-project-sub: internal
```

There is a problem with the indenting here.